src/vs/workbench/contrib/chat/browser/chatStatus.ts

```diff
@@ -267,8 +240,25 @@ export class ChatStatusBarEntry extends Disposable implements IWorkbenchContribu
 			command: ShowTooltipCommand,
 			showInAllWindows: true,
 			kind,
-			tooltip: { element: (token: CancellationToken) => this.dashboard.value.show(token) }
-		};
+			tooltip: {
+				element: (token: CancellationToken) => {
+					const store = new DisposableStore();
+					store.add(token.onCancellationRequested(() => {
+						store.dispose();
+					}));
+					const elem = ChatStatusDashboard.instantiateInContents(this.instantiationService, store);
+
+					// todo@connor4312/@benibenj: workaround for #257923
+					store.add(disposableWindowInterval(mainWindow, () => {
```

Comment:
Can we please avoid a hack like this, why is it now needed when it was not before?